.clang-tidy

```diff
@@ -39,8 +39,7 @@ Checks: '
          -readability-use-anyofallof,
          -misc-no-recursion,
          -bugprone-easily-swappable-parameters,
-         -bugprone-suspicious-include,
-         -bugprone-unchecked-optional-access
```

Comment:
Did you try this, e.g., on the reference branch? I remember a lot of false positives on Hyrise a year ago or so.